src/meta/src/hummock/manager/time_travel.rs

```diff
@@ -275,9 +277,19 @@ impl HummockManager {
     /// The version is retrieved from `hummock_epoch_to_version`, selecting the entry with the largest epoch that's lte `query_epoch`.
     ///
     /// The resulted version is complete, i.e. with correct `SstableInfo`.
-    pub async fn epoch_to_version(&self, query_epoch: HummockEpoch) -> Result<HummockVersion> {
+    pub async fn epoch_to_version(
+        &self,
+        query_epoch: HummockEpoch,
+        table_id: u32,
+    ) -> Result<HummockVersion> {
         let sql_store = self.sql_store().ok_or_else(require_sql_meta_store_err)?;
         let epoch_to_version = hummock_epoch_to_version::Entity::find()
+            .filter(
+                Condition::any()
+                    .add(hummock_epoch_to_version::Column::TableId.eq(i64::from(table_id)))
+                    // for backward compatibility
```

Comment:
Is it possible for us to refill the correct table id when meta node is starting so that we don't have to ensure backward compatibility here?